step-generation/src/types.ts

```diff
@@ -375,6 +375,8 @@ export type DistributeArgs = SharedTransferLikeArgs & {
 
   /** Disposal volume is added to the volume of the first aspirate of each asp-asp-disp cycle */
   disposalVolume: number | null | undefined
+  /** Volume to condition the tip with during aspiration sequence */
+  conditioningVolume: number | null | undefined
```

Comment:
if there is no specific reason

```suggestion
    dispenseDelaySeconds?: number | null
```

Reply:
Just pattern with the other fields. They can probably all be fixed